lib/interfaces/common.js

```diff
@@ -64,15 +67,60 @@ module.exports = function(suites, context) {
 
     suite: {
       /**
-       * Exclusive suite.
+       * Create an exclusive Suite; convenience function
+       * See docstring for create() below.
        *
-       * @param {Object} mocha
-       * @param {Function} suite
+       * @param {Object} opts
+       * @returns {Suite}
        */
-
-      only: function(mocha, suite) {
-        suite.isOnly = true;
+      only: function only(opts) {
         mocha.options.hasOnly = true;
+        opts.isOnly = true;
+        return this.create(opts);
+      },
+
+      /**
+       * Create a Suite, but skip it; convenience function
+       * See docstring for create() below.
+       *
+       * @param {Object} opts
+       * @returns {Suite}
+       */
+      skip: function skip(opts) {
+        opts.pending = true;
+        return this.create(opts);
+      },
+
+      /**
+       * Creates a suite.
+       * @param {Object} opts Options
+       * @param {string} opts.title Title of Suite
+       * @param {Function} [opts.fn] Suite Function (not always applicable)
+       * @param {boolean} [opts.pending] Is Suite pending?
+       * @param {string} [opts.file] Filepath where this Suite resides
+       * @param {boolean} [opts.isOnly] Is Suite exclusive?
+       * @returns {Suite}
+       */
+      create: function create(opts) {
+        var suite = Suite.create(suites[0], opts.title);
+        suite.pending = Boolean(opts.pending);
+        suite.file = opts.file;
+        suites.unshift(suite);
+        // I should be pilloried for the following.
```

Comment:
The general idea is that we don't flip a bit on a `Suite` if something like `describe.only()` is used; instead we just reference its `tests` prop to `onlyTests`, so any tests in it (and within child `Suite`s!) will be in the bucket.  

If a `Test` would override this behavior, we [destroy the reference](https://github.com/mochajs/mocha/pull/2387/files#diff-b756b5d38e193a4ff9b810229dbc9b60R139).